### DISCUSSION TITLE
`1.6.2` release (specifically for paypal)

### DISCUSSION THREAD

**Discussion by cachho** [InvoicePlane/InvoicePlane#1092] [Future Versions]
I saw that you merged the return of the Paypal payment feature for version `1.6.2` (#1046).

Is this version actually getting a release? If yes, when? Considering there hasn't been a release in almost a year.

I know I could build it myself, but I'm using docker-compose and I would prefer the comfort. Thanks.

**Comment by naui95** [comment]
Hi @cachho, thank you for the question. The version is getting a beta release in about one month. We still got a couple of issues to work on to have release ready (see milestone: https://github.com/InvoicePlane/InvoicePlane/milestone/11), but we are almost there.

As for the frequency of the updates, we have been discussing the matter and will soon release some more information about InvoicePlane and updates. This will be valid at least until we get to a release of v2.

**Comment by naui95** [comment]
Hi @cachho, FYI we released the beta version of InvoicePlane v1.6.2 that includes the payment update. You can download the latest versions from https://invoiceplane.com/downloads
